juno/assetlist.json

```diff
@@ -1427,7 +1427,50 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.svg"
+             }
+  },
+  {
+    "description": "Propose. Vote. Build.",
+    "type_asset": "cw20",
+    "address": "juno12mcwmd6wqhledkjsurlfqtc8j0pedvxlcxw3gs4kh2qf808ehehsen8nmw",
+    "denom_units": [
+      {
+        "denom": "cw20:juno12mcwmd6wqhledkjsurlfqtc8j0pedvxlcxw3gs4kh2qf808ehehsen8nmw",
+        "exponent": 0
+      },
+      {
+        "denom": "yfd",
+        "exponent": 6
+      }
+    ],
+    "base": "cw20:juno12mcwmd6wqhledkjsurlfqtc8j0pedvxlcxw3gs4kh2qf808ehehsen8nmw",
+    "name": "Y-Foundry DAO",
+    "display": "yfd",
+    "symbol": "YFD",
+    "logo_URIs": {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.png",
+      "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/yfd.svg"
+    }
+  },
+  {
+    "denom_units": [
+      {
+        "denom": "factory/juno1qly4zcmzr2gyxtze5yt9chv2srczwwunppxjfh/NEXX",
+        "exponent": 0
+      },
+      {
+        "denom": "unexx",
+        "exponent": 6
       }
+    ],
+    "address": "juno1qly4zcmzr2gyxtze5yt9chv2srczwwunppxjfh",
+    "base": "factory/juno1qly4zcmzr2gyxtze5yt9chv2srczwwunppxjfh/NEXX",
+    "name": "NEXX GEN AI",
+    "display": "nexx",
+    "symbol": "NEXX",
+    "logo_URIs": {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/nexx.png"
```

Comment:
I don't see the logo images for yfd and nexx being uploaded. Could those be included as well, please?